modules/layers/src/text-layer/text-layer.js

```diff
@@ -36,14 +36,33 @@ const ALIGNMENT_BASELINE = {
 
 const DEFAULT_FONT_FAMILY = 'Monaco, monospace';
 const DEFAULT_COLOR = [0, 0, 0, 255];
+const DEFAULT_FONT_SIZE = 64;
 
 const MISSING_CHAR_WIDTH = 32;
 
+const fontAtlasProps = [
+  'sdf',
+  'fontSize',
+  'buffer',
+  'radius',
+  'cutoff',
+  'fontFamily',
+  'fontWeight',
+  'characterSet'
+];
+
 const defaultProps = {
   fp64: false,
   sizeScale: 1,
-  fontFamily: DEFAULT_FONT_FAMILY,
+  // used in `makeFontAtlas`
+  sdf: false,
   characterSet: DEFAULT_CHAR_SET,
+  fontFamily: DEFAULT_FONT_FAMILY,
+  fontSize: DEFAULT_FONT_SIZE,
```

Comment:
- `fontSize` will be very confusing as we also have `getSize` and `sizeScale`.
- We could allow `sdf` be either a boolean or an object. The user can specify SDF generation options by supplying `sdf: {fontSize, buffer, ...}`.